generate.js

```diff
@@ -1,3 +1,5 @@
+// jshint esversion:6
```

Comment:
```suggestion
```

Reply:
I resolved that, thank you !! I added because it was giving me error symbols.

Reply:
I don't know how to revert json-lock file.